ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -1019,15 +1026,15 @@ def verify_noobaa_endpoint_count():
     """
     ocs_version = version.get_semantic_ocs_version_from_config()
     disable_noobaa = config.COMPONENTS["disable_noobaa"]
-    managed_service = (
-        config.ENV_DATA["platform"].lower() in constants.MANAGED_SERVICE_PLATFORMS
+    hci_managed_service = (
+        config.ENV_DATA["platform"].lower() in constants.HCI_PC_OR_MS_PLATFORM
     )
     max_eps = (
         constants.MAX_NB_ENDPOINT_COUNT if ocs_version >= version.VERSION_4_6 else 1
     )
     if config.ENV_DATA.get("platform") == constants.IBM_POWER_PLATFORM:
         max_eps = 1
-    if not (disable_noobaa or managed_service):
+    if not (disable_noobaa or hci_managed_service):
```

Comment:
@ebondare we will be enabling noobaa configuration in Phase II. is this something that will need a change post that?

Reply:
Yes